patroni/postgresql.py

```diff
@@ -1563,50 +1555,70 @@ def postmaster_start_time(self):
         except psycopg2.Error:
             return None
 
+    def drop_replication_slot(self, name):
+        cursor = self._query(('SELECT pg_drop_replication_slot(%s) WHERE EXISTS (SELECT 1 ' +
+                              'FROM pg_replication_slots WHERE slot_name = %s AND NOT active)'), name, name)
+        # In normal situation rowcount should be 1, otherwise either slot doesn't exists or it is still active
+        return cursor.rowcount == 1
+
+    @staticmethod
+    def compare_slots(s1, s2):
```

Comment:
Judging from `load_replication_slots` we can have only:
```python
{ slot1: { "type": "physical" } } 
```
or
```python
{ slot2: { "type": "physical", "plugin": "plugin", "database": db } }
```

Then doesn't it mean that we can just use `s1 == s2` here?